src/tasks/buildLocal.ts

```diff
@@ -19,10 +20,12 @@ const buildLocal = async (context: vscode.ExtensionContext) => {
 
     const CMakeLists = readFileSync(
         join(folders[0].uri.fsPath, "./CMakeLists.txt"),
-        "utf-8"
+        "utf-8" // I don't think this is required
     )
 
-    if (!CMakeLists.includes("#local")) {
+    if (!CMakeLists.includes("#local")) { // What is this? Why does CMakeLists include #local?
+        // I'm guessing it's because you are lazy and you didn't use an `if` in the CMakeLists.
```

Comment:
Its because i want to know if i need to delete the things in cbuild

Reply:
Why would you ever need to delete them?

It's dumb.
Have two different build directories for roborio and local.

I suggest build/local and build/roborio